.changeset/markdown-component.md

```
@@ -0,0 +1,5 @@
+---
+'@siemens/ix': minor
+---
+
+Add the `ix-markdown` component with iX styling, task-list checkboxes, registered web components, runtime data bindings, and a required renderer function. Import the bundled Comark renderer from `@siemens/ix/markdown`, or opt into explicit-language Highlight.js syntax highlighting with `@siemens/ix/markdown/highlight`.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)